### PR TITLE
Archetype

### PR DESCRIPTION
**Pull request by konstantinzolotarev**
Moved hapi archetype from `trailsjs/trails` to `trailpack-hapi` think it better place to have them.

### CODE REVIEW

File: archetype/api/policies/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = {};
```
Comment (tjwebb) [inline archetype/api/policies/index.js]: we're omitting semi colons :)

**Comment by listepo** [comment]
+1

**Comment by tjwebb** [comment]
@konstantinzolotarev is there a similar PR which removes the stuff from the core archetype?

**Comment by konstantinzolotarev** [comment]
@tjwebb Here: trailsjs/trails#46

File: archetype/api/controllers/ViewController.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+
+  helloWorld (request, reply) {
+    reply('Hello Trails.js !')
```
Comment (tjwebb) [inline archetype/api/controllers/ViewController.js:6]: I think unless a view is being rendered, we always want to reply in JSON format.

Comment (tjwebb) [inline archetype/api/controllers/ViewController.js:6]: I guess this is a view, after all.

**Pull request merged**